src/modules/iasql_functions/rpcs/iasql_install.ts

```diff
@@ -16,7 +16,14 @@ export class IasqlInstall extends RpcBase {
     ctx: Context,
     ...params: string[]
   ): Promise<RpcResponseObject<typeof this.outputTable>[]> => {
-    await iasql.install(params, dbId, dbUser, false, false, ctx);
+    await iasql.maybeOpenTransaction(ctx.orm);
```

Comment:
My first question would be to which pl/pgsql functions should this belong? All the install/commit/etc are generic rpcs. Either they need a new structure and not use the generic RPC or we add some special casing in there. I feel there's something I'm missing out on.

Reply:
So the simplest solution is rename it to something like `iasql_inner_install` (`iasqlInnerInstall` in the module code) and then add to the `after_install.sql` a simple `pl/pgsql` named `iasql_install` that does this check (and waits instead of immediately fails) and then calls `iasql_inner_install` with the same input arguments once ready.

Then similarly the functions that need to fail when there's an ongoing transaction can have that logic in place, instead.